src/OSPSuite.Presentation/Presenters/Charts/ChartDisplayPresenter.cs

```diff
@@ -404,14 +404,11 @@ private void updateChart()
 
             bindChartToView();
 
-            RefreshAxisBinders();
-
             //maybe can be done better
             View.ReOrderLegend();
          }
          updateViewLayout();
 
-         //TODO WHY DO WE NEED IT TWICE ? 
          RefreshAxisBinders();
```

Comment:
This has to be called outside the updater because when resetting the axis visual range, first the axis whole range is set to auto, then it's automatically calculated extents are used to set the visual range.

When we perform the refresh inside the updater, the extents are not calculated it seems, so we always set the visual range min to 0. This is corrected for log scale to Float.Epsilon.